src/core/controllers/orbit-state.js

```diff
@@ -147,19 +136,25 @@ export default class OrbitState {
    * @param {[Number, Number]} pos - position on screen where the pointer is
    */
   pan({pos, startPos}) {
+    if (this._interactiveState.isRotating) {
```

Comment:
Don't have to address in this PR.
Quesiton: Can we make the decision of panning/rotating before pan()/rotate() gets called so we don't need these if conditions?

Reply:
I like that, too. The logic is inside ViewControls which will impact all the viewports, and ViewContols is under refactor, need discuss before the change. That's why I didn't do that.